inspectit-ocelot-documentation/docs/enduser-monitoring/eum-server-configuration.md

```diff
@@ -466,21 +466,21 @@ inspectit-eum-server:
 
 ### Metrics
 
-The EUM server comes with the same Prometheus and InfluxDB exporter as the Ocelot agent.
+The EUM server comes with the same Prometheus, InfluxDB and OTLP metrics exporter as the Ocelot agent.
 The exporter's configurations options are the same as for the [agent](metrics/metric-exporters.md).
 However, they are located under the `inspectit-eum-server.exporters.metrics` configuration path.
 
 #### Prometheus
-By default, the prometheus exporter is enabled and available on port `8888`.
+By default, the prometheus exporter is disabled.
 
-The following configuration snippet shows the default configuration of the prometheus-exporter:
+The following configuration snippet shows the default configuration of the prometheus-exporter, exposing on port `8888`:
 ```YAML
 inspectit-eum-server:
   exporters:
     metrics:
       prometheus:
         # Determines whether the prometheus exporter is enabled.
-        enabled: true
+        enabled: ENABLED
```

Comment:
Maybe I misunderstood something, but you say earlier that by default prometheus exporter is disabled (l. 474)
This however suggests, that it is enabled. Or am I mistaken? :)